### ISSUE TITLE
Dependency package outdated?

### ISSUE THREAD

**Issue by Azurba** [llfbandit/app_links#117]
Hi! I am trying to use the app_links on a Flutter desktop (Windows) as a recommendation from OAuth2 package in an attempt to implement a redirect function, and on the documentation "READ ME" it is suggested to use the win32_registry package. However in the example code, the package 
url_protocol is being used to use the registerProtocolHandler on the _registerWindowsProtocol() function which is used to "Register our protocol only on the Windows platform"

Upon trying to get the package url_protocol, it has a dependency conflict with the win32_registry.

```
Because every version of url_protocol depends on win32 ^2.0.5 and win32_registry >=1.1.3 depends on win32 ^5.4.0,
  url_protocol is incompatible with win32_registry >=1.1.3.
So, because app_test depends on both win32_registry ^1.1.3 and url_protocol any, version solving failed.
[id_cards.pdf](https://github.com/llfbandit/app_links/files/15100066/id_cards.pdf)

```

Is there any alternative to go around this issue or am I missing anything on the document and am doing anything wrong?

Thank you so much!

**Does it related to**

[ ] App Links (Android)  
[X] Deep Links (Android)  
[ ] Universal Links (iOS)  
[ ] or Custom URL schemes? (iOS)  

**Does the example project work?**

[ ] Yes  
[X] No  
[ ] Irrelevant here  

**Did you fully read the instructions for the targeted platform before submitting this issue?**
Yes, but I might be missing something?
Uploaded your files to webserver, HTTPS, direct connection, scheme pattern setup, ...

[ ] Yes  
[ ] No  
[X] Irrelevant here
![issue](https://github.com/llfbandit/app_links/assets/58566178/41174274-c170-4cb7-b60f-b764f4c82c0b)

**Comment by thanhle7** [comment]
Hi,
Try removing package url_protocol then making a clean build should be fine.

**Comment by llfbandit** [comment]
Where did you see documentation stated to add `url_protocol` package?
You shouldn't have to do anything to make the example work.

**Comment by Azurba** [comment]
@thanhle7 unfortunately using the flutter clean didn't work for me.

**Comment by Azurba** [comment]
@llfbandit on teh example there is an " import 'url_protocol/api.dart'; ". On a clean project, I copied and pasted the example and the registerProtocolHandler, unregisterProtocolHandler and the import got errors (. When I searched for the url_protocol package, these functions are defined there. So I assumed I had to get this package (function isn't defined and target url URI is not defined) as well.
![url_protocol](https://github.com/llfbandit/app_links/assets/58566178/d2105f0a-f054-42f4-861e-c3c3065b8914)
